contentful/extensions/ad-tags/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2016, Contentful
```

Comment:
This is based off of Contentful code at https://github.com/contentful/extensions/tree/master/samples/json-form-editor and it requires us to include their copyright. So here it is.